doc/api/http.md

```diff
@@ -1539,6 +1539,8 @@ convenience method. The only difference between this method and
 automatically. Note that response data must be consumed in the callback
 for reasons stated in [`http.ClientRequest`][] section.
 
+Note: The `options` properties are copied via `util._extend()` which doesn't include the prototype.
```

Comment:
Couple of style nits:

1. Please wrap at 80 chars
2. `s/Note:/*Note*:/`
3. Rewording to something like the following is likely sufficient:

```
*Note*: The `options` object's "own" properties are copied when passed.
```

The fact that it uses `util._extend()` is an internal implementation detail that should not be worth documenting and could change at any point in the future)